smartapps/smartthings/ecobee-connect.src/ecobee-connect.groovy

```diff
@@ -595,54 +562,26 @@ private refreshAuthToken() {
 	if(!atomicState.refreshToken) {
 		log.warn "Can not refresh OAuth token since there is no refreshToken stored"
 	} else {
-
 		def refreshParams = [
-				method: 'POST',
-				uri   : apiEndpoint,
-				path  : "/token",
-				query : [grant_type: 'refresh_token', code: "${atomicState.refreshToken}", client_id: smartThingsClientId],
+			method: 'POST',
+			uri   : apiEndpoint,
+			path  : "/token",
+			query : [grant_type: 'refresh_token', code: "${atomicState.refreshToken}", client_id: smartThingsClientId],
 		]
 
-		log.debug refreshParams
+		log.debug "making refresh request with paramaters: $refreshParams"
```

Comment:
It probably doesn't matter, but I'm not super enthusiastic about logging refresh tokens or auth tokens